File: src/Ejercicio8/Ejercicio8.java
package Ejercicio8;

import java.util.ArrayList;
import java.util.List;
import java.util.Scanner;

public class Ejercicio8 {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        System.out.print("Ingrese una cadena: ");
        String cadena = sc.nextLine().toLowerCase();
        int totalVocales = 0;
        String [] vocales = {"a","e", "i", "o", "u"};

        for (int i = 0; i <= cadena.length() -1; i++) {
            for (int j = 0; j <= vocales.length - 1; j++) {
                if (cadena.split("")[i].equals(vocales[j])) {
                    totalVocales++;
                }
            }
        }

        System.out.println("Total vocales: " + totalVocales);

    }
}

File: src/Ejercicio8/Ejercicio8.kt
package Ejercicio8

fun main(){

    print("Ingrese una cadena: ")
    val cadena = readln().lowercase()
    var contador = 0
    val vocales = listOf<String>("a","e","i","o","u")

    for (letra in cadena){

        if (vocales.contains(letra.toString())){
            contador++
        }

    }

    println("Total vocales: $vocales")
}